omnigibson/examples/robots/curobo_example.py

```diff
@@ -0,0 +1,400 @@
+    attached_obj = {robot.eef_link_names["left"]: cologne.root_link}
+    attached_obj_scale = {robot.eef_link_names["left"]: cologne_scale}
+    plan_and_execute_trajectory(
+        cmg, target_pos, target_quat, emb_sel, attached_obj, attached_obj_scale, eef_markers, env, robot
+    )
+
+    # Grasp fridge door (right hand)
+    right_hand_pos, right_hand_quat = T.pose_transform(*fridge.get_position_orientation(), *fridge_door_local_pose)
+    target_pos = {robot.eef_link_names["right"]: right_hand_pos}
+    target_quat = {robot.eef_link_names["right"]: right_hand_quat}
+    emb_sel = CuroboEmbodimentSelection.ARM
+    attached_obj = {robot.eef_link_names["left"]: cologne.root_link}
+    attached_obj_scale = {robot.eef_link_names["left"]: cologne_scale}
+    plan_and_execute_trajectory(
+        cmg, target_pos, target_quat, emb_sel, attached_obj, attached_obj_scale, eef_markers, env, robot
+    )
+    attached_obj = {
+        robot.eef_link_names["left"]: cologne.root_link,
+        robot.eef_link_names["right"]: fridge.links["link_0"],
+    }
+    control_gripper(env, robot, attached_obj)
+    assert robot._ag_obj_in_hand["left"] == cologne and robot._ag_obj_in_hand["right"] == fridge
+
+    # Pull fridge door (right hand)
+    right_hand_pos, right_hand_quat = T.pose_transform(*fridge.get_position_orientation(), *fridge_door_open_local_pose)
+    target_pos = {robot.eef_link_names["right"]: right_hand_pos}
+    target_quat = {robot.eef_link_names["right"]: right_hand_quat}
+    emb_sel = CuroboEmbodimentSelection.DEFAULT
+    attached_obj = {
+        robot.eef_link_names["left"]: cologne.root_link,
+        robot.eef_link_names["right"]: fridge.links["link_0"],
+    }
+    attached_obj_scale = {robot.eef_link_names["left"]: cologne_scale, robot.eef_link_names["right"]: fridge_door_scale}
+    plan_and_execute_trajectory(
+        cmg, target_pos, target_quat, emb_sel, attached_obj, attached_obj_scale, eef_markers, env, robot
+    )
+    attached_obj = {robot.eef_link_names["left"]: cologne.root_link}
+    control_gripper(env, robot, attached_obj)
+    assert robot._ag_obj_in_hand["left"] == cologne and robot._ag_obj_in_hand["right"] == None
+
+    # Place the cologne (left hand)
+    left_hand_pos, left_hand_quat = T.pose_transform(*fridge.get_position_orientation(), *fridge_place_local_pose)
+    target_pos = {robot.eef_link_names["left"]: left_hand_pos}
+    target_quat = {robot.eef_link_names["left"]: left_hand_quat}
+    emb_sel = CuroboEmbodimentSelection.DEFAULT
+    attached_obj = {robot.eef_link_names["left"]: cologne.root_link}
+    attached_obj_scale = {robot.eef_link_names["left"]: cologne_scale}
+    plan_and_execute_trajectory(
+        cmg, target_pos, target_quat, emb_sel, attached_obj, attached_obj_scale, eef_markers, env, robot
+    )
+    attached_obj = None
+    control_gripper(env, robot, attached_obj)
+    assert robot._ag_obj_in_hand["left"] == None and robot._ag_obj_in_hand["right"] == None
+
+    og.shutdown()
```

Comment:
should this shutdown be here if all the tests are running? or is this guaranteed to be the final test?